man/tpm2_policynamehash.1.md

```diff
@@ -0,0 +1,49 @@
+% tpm2_policynamehash(1) tpm2-tools | General Commands Manual
+
+# NAME
+
+**tpm2_policynamehash**(1) - Evaluates to an authorization policy tied to
```

Comment:
This sentence seems awkward. Is this sentence the same as, "Couples a policy with a specific name of an object", is that what you're trying to say here?